src/main/resources/userEmailTemplate-userRegisterForCourse.html

```diff
@@ -0,0 +1,14 @@
+<p>Hello, ${userName}</p>
+
+<p>Thank you for registering your Google ID with TEAMMATES!</p>
```

Comment:
@damithc Since the template is common for both instructors and students, had to add in a new placeholder for the "an instructor" or "a student" bit of the above suggestion. Also, then added a seperate test cases for student and instructor emails and renamed test files accordingly.